src/main/java/org/spongepowered/api/CatalogType.java

```diff
@@ -61,4 +67,14 @@
      */
     String getName();
 
+    @Override
+    default Text toText() {
```

Comment:
This is extremely specific and unmodifiable. If plugins want to represent a `CatalogType` as `Text`, they'll do so however they please. Making this a contracted method for all `CatalogType`s is absolutely a "no-go".